apps/simple-trading-app/src/app/components/nav/nav.tsx

```diff
@@ -11,20 +11,22 @@ interface NavProps {
 export const Nav = ({ className, tabs = false }: NavProps) => {
   return (
     <nav role={tabs ? 'tablist' : 'menu'} className={className}>
-      {routerConfig.map((r) => (
-        <NavLink
-          role={tabs ? 'tab' : 'menuitem'}
-          key={r.name}
-          to={r.path}
-          className={({ isActive }) =>
-            `text-h5 block md:mb-40 px-40 md:text-black md:dark:text-white ${
-              isActive && 'text-white md:text-blue md:dark:text-blue'
-            }`
-          }
-        >
-          <NavItem iconName={r.icon} label={r.text} />
-        </NavLink>
-      ))}
+      {routerConfig.map((r) =>
+        r.icon ? (
```

Comment:
question (blocking?): this reads really strange. if icon is required to render nav links, why not make it a non-optional property? not sure if it makes sense to have route items with missing icons, if they don't render, or is there a specific use-case for those? if that is the case, i'd add a new prop on the route item so it's understandable why they are not rendering, like `isIncludedInNavigation`, or something similar.

Reply:
home page (with no icon) doesn't need to be displayed in the 'iconic' mobile menu. I wouldn't create new special property for matching it. Come on, it's trivial, clear case, understandable at a glance

Reply:
i understand, what i'm saying is rendering something on the condition whether it has an icon or not doesn't seem to me like the correct abstraction, it works in this case because of a coincidence, and makes future refactor more difficult. i'm happy with it if people agree to do it this way, feel free to resolve this and merge

Reply:
ok, now it's improved according to your indications above